utils/DeployedContractsHelpers.ts

```diff
@@ -379,6 +379,10 @@ export async function getContratAddress(hre: HardhatRuntimeEnvironment, contract
     return constants.wETH_address[hre.network.name];
   } else if (contractName === "WBTC") {
     return constants.wBTC_address[hre.network.name];
+  } else if (contractName === "XUSD" && hre.network.name == "rsk") {
```

Comment:
```suggestion
  } else if (contractName === SUPPLEMENTARY_USD_STABLE_NAME["rsk"]) {
```

Reply:
you can either do that for both rsk and mainnetFork or going with this generic approach:

```
else if (contractName === SUPPLEMENTARY_USD_STABLE_NAME[hre.network.name]) {
    return SUPPLEMENTARY_USD_STABLE_ADDRESS[hre.network.name]
}
```

Reply:
very good point, thanks